docs/word/add-headers-on-document-open.md

```diff
@@ -0,0 +1,218 @@
+---
+title: Add headers when a document opens
+description: Learn how to develop a Word add-in that implements event-based activation to add headers when a document is opened.
+ms.date: 07/02/2025
+ms.topic: how-to
+ms.localizationpriority: medium
+---
+
+# Add headers when a document opens
+
+The following sections walk you through how to develop a Word add-in that automatically changes the document header when a new or existing document opens. While this specific sample is for Word, the manifest configuration is the same for Excel and PowerPoint.
+
```

Comment:
Consider adding a link to [Activate add-ins with events](https://learn.microsoft.com/office/dev/add-ins/develop/event-based-activation) somewhere in the doc.